README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/erseco/civxd.svg?branch=master)](https://travis-ci.org/erseco/civxd)
```

Comment:
This would need to be updated prior to merging

Reply:
Solved!